core/src/apps/stellar/operations/serialize.py

```diff
@@ -68,6 +69,15 @@ def write_manage_offer_op(w, msg: StellarManageOfferOp):
     writers.write_uint64(w, msg.offer_id)
 
 
+def write_manage_buy_offer_op(w, msg: StellarManageBuyOfferOp):
```

Comment:
please reuse `write_manage_offer_op` instead

Reply:
I'm waiting for your response about `amount` so I can decide how to rewrite that one.